frontend/components/TextArea/TextArea.js

```diff
@@ -0,0 +1,30 @@
+import React, { forwardRef } from "react";
+import styled from "@emotion/styled";
+import Text from "../Text/Text";
+import { color, fontWeight, spacing } from "../../theme";
+
+const StyledLabel = styled.label`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledTextArea = styled.textarea`
+  border-radius: ${spacing["0.5"]};
+  padding: ${spacing["0.5"]} ${spacing["0.75"]};
+  font-family: "IBM Plex Sans", sans-serif;
+`;
+
+const TextArea = forwardRef(
+  ({ id, labelText, errorMessage, ...props }, ref) => {
+    return (
+      <StyledLabel htmlFor={id}>
+        <Text fontWeight={fontWeight.bold}>{labelText}</Text>
+        <StyledTextArea id={id} ref={ref} {...props} />
+        {errorMessage && <Text color={color.red}>{errorMessage}</Text>}
+      </StyledLabel>
+    );
+  }
+);
+TextArea.displayName = "TextArea";
```

Comment:
Please elaborate on why this is necessary

Reply:
DisplayName is a property used by react for error handling on the development side. If there was an error in the TextArea component, the string "TextArea" would be used to identify where the error occurred.